MapPointMapper/ViewController.swift

```diff
@@ -203,9 +211,20 @@ class ViewController: NSViewController, MKMapViewDelegate, NSTextFieldDelegate {
 
   - parameter input: `NSString` to parse and draw on the map. If no string is given this is essentially a noop
```

Comment:
probably wanna update docs with return value now (its private so importance is meh)